dpnp/dpnp_iface_arraycreation.py

```diff
@@ -894,79 +899,67 @@ def diag(v, /, k=0, *, device=None, usm_type=None, sycl_queue=None):
     --------
     :obj:`diagonal` : Return specified diagonals.
     :obj:`diagflat` : Create a 2-D array with the flattened input as a diagonal.
-    :obj:`trace` : Return sum along diagonals.
-    :obj:`triu` : Return upper triangle of an array.
-    :obj:`tril` : Return lower triangle of an array.
+    :obj:`trace` : Return the sum along diagonals of the array.
+    :obj:`triu` : Upper triangle of an array.
+    :obj:`tril` : Lower triangle of an array.
 
     Examples
     --------
     >>> import dpnp as np
-    >>> x0 = np.arange(9).reshape((3, 3))
-    >>> x0
+    >>> x = np.arange(9).reshape((3, 3))
+    >>> x
     array([[0, 1, 2],
            [3, 4, 5],
            [6, 7, 8]])
 
-    >>> np.diag(x0)
+    >>> np.diag(x)
     array([0, 4, 8])
-    >>> np.diag(x0, k=1)
+    >>> np.diag(x, k=1)
     array([1, 5])
-    >>> np.diag(x0, k=-1)
+    >>> np.diag(x, k=-1)
     array([3, 7])
 
-    >>> np.diag(np.diag(x0))
+    >>> np.diag(np.diag(x))
     array([[0, 0, 0],
            [0, 4, 0],
            [0, 0, 8]])
 
     Creating an array on a different device or with a specified usm_type
 
-    >>> x = np.diag(x0) # default case
-    >>> x, x.device, x.usm_type
+    >>> res = np.diag(x) # default case
+    >>> res, res.device, res.usm_type
     (array([0, 4, 8]), Device(level_zero:gpu:0), 'device')
 
-    >>> y = np.diag(x0, device="cpu")
-    >>> y, y.device, y.usm_type
+    >>> res_cpu = np.diag(x, device="cpu")
+    >>> res_cpu, res_cpu.device, res_cpu.usm_type
     (array([0, 4, 8]), Device(opencl:cpu:0), 'device')
 
-    >>> z = np.diag(x0, usm_type="host")
-    >>> z, z.device, z.usm_type
+    >>> res_host = np.diag(x, usm_type="host")
+    >>> res_host, res_host.device, res_host.usm_type
     (array([0, 4, 8]), Device(level_zero:gpu:0), 'host')
 
```

Comment:
`See Also` link is missing from `dpnp.eye` description to `diag` function

Reply:
Thank you for noticing that!
I added it.